tests/_support/Step/Acceptance/ShopSystem/GenericShopSystemStep.php

```diff
@@ -265,4 +276,28 @@ public function getActingPaymentMethod($paymentMethod): string
         }
         return $paymentMethod;
     }
+
+    public function logInToAdministrationPanel()
+    {
+        $this->amOnPage($this->getLocator()->page->admin_login);
+        try {
+            $this->preparedFillField(
+                $this->getLocator()->wordpress_sign_in->user,
+                $this->getCustomer(static::ADMIN_USER)->getEmailAddress(),
+                10
+            );
+            $this->preparedFillField(
+                $this->getLocator()->wordpress_sign_in->pass,
+                $this->getCustomer(static::ADMIN_USER)->getPassword()
+            );
+            $this->preparedClick($this->getLocator()->wordpress_sign_in->login, 60);
+        } catch (NoSuchElementException $e) {
+            $this->amOnPage($this->getLocator()->page->admin_login);
+        }
+    }
+
+    public function validateWpAdminPage(): void
```

Comment:
Is this function redundant? I can't find where it is used

Reply:
Good catch. I initially created this function, but decided to go with different approach.